test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -219,11 +219,12 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 		<exec executable="${JAVA_COMMAND}" failonerror="true">
 			<arg line="${DUMP_OPTION}" />
-			<arg line="-Xitsn1000" />
+			<arg value="-Xitsn1000" />
```

Comment:
`line` splits the string based on embedded whitespace. It makes no practical difference here because the string doesn't have any spaces, but I think it's clearer this way: the string will form exactly one argument on the command line.